openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2558,6 +2558,8 @@ The following is a non-normative example of a Wallet Attestation:
 
 To use the Wallet Attestation towards the Authorization Server, the Wallet MUST generate a proof of possession according to Section 5.2 "Client Attestation PoP JWT" of Attestation-Based Client Authentication.
 
+The `sub` claim of the Wallet Attestation JWT is picked by the Wallet Provider and represents the `client_id` of the Wallet. This value should be shared by all Wallet Instances from that Wallet Provider for privacy reasons.
```

Comment:
should this be normative (SHOULD)?

Reply:
```suggestion
The `sub` claim of the Wallet Attestation JWT is picked by the Wallet Provider and represents the `client_id` of the Wallet. For privacy reasons, this value is the same across Wallet Instances of that Wallet Provider.
```

I would honestly, add a short privacy considerations section about this and point to it from here